yast-travis-ruby

```diff
@@ -279,4 +279,5 @@ rpmbuild -bb --nocheck package/*.spec || rpmbuild -bb --nocheck --nodeps package
 rpm -iv --force --nodeps $PKG_DIR/RPMS/*/*.rpm
 rpm -Uv --force --nodeps $PKG_DIR/RPMS/*/*.rpm
 # get the plain package names and remove all packages at once
-rpm -ev --nodeps `rpm -q --qf '%{NAME} ' -p $PKG_DIR/RPMS/*/*.rpm`
+mapfile -t packages < <(rpm -q --qf '%{NAME} ' -p $PKG_DIR/RPMS/*/*.rpm)
```

Comment:
Why do you need to use mapfile here?

Reply:
See https://github.com/koalaman/shellcheck/wiki/SC2046 and https://github.com/koalaman/shellcheck/wiki/SC2207, we need to prevent from the quoting issue but still need the separate files as arguments so we need mapfile to create an array.

It was rather a theoretical issue, unless the package path contains space (or any other tricky character) the old code worked without problems. This was not a real problem.